pkgs/development/ocaml-modules/ocplib-endian/default.nix

```diff
@@ -1,24 +1,23 @@
-{ stdenv, lib, fetchzip, ocaml, findlib, ocamlbuild, cppo }:
+{ lib, buildDunePackage, fetchzip, cppo }:
 
-let version = "1.0"; in
-
-stdenv.mkDerivation {
-  name = "ocaml${ocaml.version}-ocplib-endian-${version}";
+buildDunePackage rec {
+  version = "1.1";
+  pname = "ocplib-endian";
 
   src = fetchzip {
     url = "https://github.com/OCamlPro/ocplib-endian/archive/${version}.tar.gz";
-    sha256 = "0s1ld3kavz892b8awyxyg1mr98h2g61gy9ci5v6yb49bsii6wicw";
+    sha256 = "sha256-zKsSkhlZBXSqPtw+/WN3pwo9plM9rDZfMbGVfosqb10=";
   };
 
-  buildInputs = [ ocaml findlib ocamlbuild cppo ];
+  minimalOCamlVersion = "4.02.3";
```

Comment:
This is what the name of the attribute ought to be. Unfortunately, it has been chosen to spell it `minimumOCamlVersion`.

Moreover, such a low minimum is equivalent to the absence of minimum. I therefore suggest to remove this trivial constraint.

```suggestion
```

Reply:
Done